src/layouts/_proxied-dot-io/consul/index.tsx

```diff
@@ -51,21 +51,23 @@ function ConsulIoLayout({ children, data }: Props): React.ReactElement {
 					<ProductSubnav
 						menuItems={[
 							{ text: 'Overview', url: '/' },
-							{
-								text: 'Use Cases',
-								submenu: [
-									{
-										text: 'Consul on Kubernetes',
-										url: '/consul-on-kubernetes',
-									},
-									...useCaseNavItems.map((item) => {
-										return {
-											text: item.text,
-											url: `/use-cases/${item.url}`,
-										}
-									}),
-								].sort((a, b) => a.text.localeCompare(b.text)),
-							},
+							consulNav.useCases.length > 0
```

Comment:
Is there any possibility `consulNav` might be undefined? this would error in that case. 💭

Reply:
I don't think so as this would return an empty array if useCases were empty https://github.com/hashicorp/dev-portal/pull/669/files#diff-dc15fdaf1595690d3b8934305132caed4e9a54c929a5e2465af1041e7d41e32aR1-R7

Reply:
```suggestion
							consulNav && consulNav.useCases?.length
```

I can do some defensive coding here though.